src/btree/bt_page.c

```diff
@@ -805,8 +794,8 @@ __page_inmem_prepare_update(WT_SESSION_IMPL *session, WT_ITEM *value, WT_CELL_UN
         if ((unpack->tw.start_ts == unpack->tw.stop_ts &&
```

Comment:
We can simplify this check to be both start and stop are prepared

Reply:
I think we might still need to keep this check for now to keep the current behaviour for attach storage.
```
if (WT_TIME_WINDOW_HAS_START_PREPARE(tw) || (unpack->tw.start_ts == unpack->tw.stop_ts &&
              unpack->tw.durable_start_ts == unpack->tw.durable_stop_ts &&
              unpack->tw.start_txn == unpack->tw.stop_txn)
```

and remove the other check once we align the code

Reply:
done